pytorch_lightning/__init__.py

```diff
@@ -4,6 +4,18 @@
 
 from pytorch_lightning.__about__ import *  # noqa: F401, F403
 
+VERBOSE = 15  # between logging.INFO and logging.DEBUG, used for logging in production use cases
```

Comment:
super nit: verbose as a log level might conflict with `verbose` flags. so if I specify `trainer.verbose_evaluate` it might lead to people thinking logs would come at this level. the issue proposed `DETAIL` which avoids this collision

Reply:
```suggestion
VERBOSE = 15  # between logging.INFO and logging.DEBUG, used for logging in batch use cases
```